aries-site/src/pages/design-tokens/overview.mdx

```diff
@@ -0,0 +1,100 @@
+
+Token names are purpose-driven, rather than hard-coded values, which provides the flexibility for token values to evolve over time with our visual style. When tokens are implemented across design and development, style updates will propagate consistently across applications simply by pulling in the latest version of the tokens.
+
+Design tokens allow us to:
+
+- Codify design decisions
+- Enable teams across various tech stacks to build efficiently and consistenly with the HPE Design System
+- Replace manual maintenance methods with faster automated processes that keep design and development synchronized
+- Build a shared language across design and development
+
+## Types of design tokens
+
+Tokens are grouped into 3 levels: [primitive](#primitive-tokens), [semantic](#semantic-tokens), and [component](#component-tokens). Each level exists to convey specificity, intended usage, and enhance token consumption and delivery. The name and intended use of a token becomes more specific as you move from primitive to semantic to component levels.
+
+<Notification
+  message={`When consuming design tokens, begin with the most specific token level available. Look first to see if a component token exists. If not available, use semantic tokens as a fallback.`}
+  status="info"
+  width={{ max: 'large' }}
+  margin={{ bottom: 'medium' }}
+/>
+
+### Primitive tokens
+
+Primitive tokens define the most foundational level of the token system, establishing fundamental attributes like color and dimension.
+
+The value of a primitive token will always be a CSS value such as a hex code, pixel, or rem value. Typically, primitive token names describe the token value and not where it should be applied.
+
+Primitive design tokens should not be used directly. They exist to provide the foundational options and be referenced in semantic and component design tokens.
+
+<Example height={{ min: 'xsmall' }}>
+  <Box align="center" direction="row" gap="small">
+    <Box background="brand" pad="medium" round="small" flex={false} />
+    <Text size="large">hpe.base.color.green.600</Text>
+  </Box>
+</Example>
+
+### Semantic tokens
+
+Semantic tokens apply a specific meaning to a primitive token to establish how and when a style should be applied. Semantic tokens capture design decisions for color, spacing, elevation, typography, and more.
+
+The value of a semantic token can either be a reference to a primitive token or another semantic token.
+
+Semantic tokens can have different values in different contexts. For example, the value of `hpe.color.background.back` might differ for light versus dark mode. Similarly, the value of `hpe.spacing.large` might differ for wide versus narrow viewports. By building with semantic tokens, the values can flex based on the context.
+
+<Example height={{ min: 'xsmall' }}>
+  <Box align="center" direction="row" gap="small">
```

Comment:
I find myself wanting a little more variety (besides color) in these examples. To address in subsequent passes.

Reply:
Good note, yes we can modify in future.